testdata/example-models/nonmem/IOVMM/IOVMM.json

```diff
@@ -0,0 +1,155 @@
+{
+	"run_details": {
+		"version": "7.4.3",
+		"run_start": "Thu Sep  5 14:38:15 EDT 2019",
+		"run_end": "Thu Sep  5 14:38:31 EDT 2019",
+		"estimation_time": 10.94,
+		"covariance_time": 0.57,
+		"function_evaluations": 367,
+		"significant_digits": 3.1,
+		"problem_text": "LEM 10 mixture model and IOV on CL",
+		"estimation_method": [
+			"First Order"
+		],
+		"data_set": "../MixSim.CSV",
+		"number_of_patients": 300,
+		"number_of_obs": 12600,
+		"number_of_data_records": 13500
+	},
+	"run_heuristics": {},
+	"parameters_data": [
+		{
+			"method": "TABLE NO.     1: First Order: Goal Function=MINIMUM VALUE OF OBJECTIVE FUNCTION: Problem=1 Subproblem=0 Superproblem1=0 Iteration1=0 Superproblem2=0 Iteration2=0",
+			"estimates": {
+				"theta": [
+					14.2204,
+					100.996,
+					0.824135,
+					0.988974,
+					1.70925
+				],
+				"omega": [
+					0.0490352,
+					0,
+					0.0482149,
+					0,
+					0,
+					0.0359252,
+					0,
+					0,
+					0,
+					0.0359252,
+					0,
+					0,
+					0,
+					0,
+					0.0359252
+				],
+				"sigma": [
+					0.0458045,
+					0
+				]
+			},
+			"std_err": {},
```

Comment:
hmm we'll need to think about this object - do we want it to be an empty object vs an object with all zeros. Basically, we might want to initialize the stderr blocks to be all zeros of the same length as the parameter values to make sure the structure is always there